meta-lmp-base/classes/image_types_ostree.bbclass

```diff
@@ -178,6 +178,7 @@ IMAGE_CMD:ostreepush () {
 }
 
 GARAGE_PUSH_RETRIES ?= "3"
+GARAGE_PUSH_RETRIES_SLEEP ?= "0"
```

Comment:
We can probably have this set to 10 by default, or similar.

Reply:
My ideia is propose this in meta-updater with the sleep disabled to don't change the current implementation.
We can change it after in our side but if upstream like to have this I can change the default to 10.

I am testing with 10 in main-next https://github.com/quaresmajose/meta-updater/commit/06d1404743008ea868812ac2b05fc3af997a3094